Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Medical/hypospray.yml

```
@@ -53,28 +53,114 @@
+    tag: AutoinjectorCartridge
 
 - type: entity
-  name: epinephrine autoinjector cartridge
+  name: autoinjector cartridge
   parent: [BaseItem, BaseRestrictedContraband]
-  description: Contains 10u of epinephrine, used in an epinephrine autoinjector.
-  id: CartridgeEpinephrine
+  description: Used in a cartridge autoinjector.
+  id: BaseCartridgeAutoinjector
   components:
   - type: Item
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)